requirements.txt

```diff
@@ -1,6 +1,6 @@
 adsputils==v1.2.7
 alembic==0.9.1
-psycopg2==2.8.3
+psycopg2-binary==2.8.3
```

Comment:
I don't know if there is a practical difference to the deployment, but it might be worth reverting this back to `psycopg2` before the merge.

Reply:
Done